codegen/type_converter.go

```diff
@@ -727,7 +814,18 @@ func (c *TypeConverter) GenStructConverter(
 		}
 	}
 
-	err := c.genStructConverter("", "", "", fromFields, toFields, fieldMap)
+	if c.isRecursiveStruct(toFields) || c.isRecursiveStruct(fromFields) {
+		c.useRecurGen = true
+	} else {
+		c.useRecurGen = false
+	}
```

Comment:
`c.useRecurGen = c.isRecursiveStruct(toFields) || c.isRecursiveStruct(fromFields)` ?

Reply:
This is how the nested generated functions disappeared.. 😅 When useRecurGen is false the code generation looks exactly like before, otherwise it's what you saw earlier (without the blank {} scopes), and it's only true if necessary, i.e. when the input or output have some recursive element